requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
I am not familiar with python, but shouldn't stripe be responsible for specifying the `requests` version in that case?